recipes/cms-combine/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cms-combine" %}
+{% set version = "10.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a6fecc8b4adc4e631425eeee7501d0258ab64ef36b24680696c2e6f8d38403e7
+  folder: HiggsAnalysis/CombinedLimit
+  patches:
+    - clang_warnings.patch
+    - makefile.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports: python
```

Comment:
If you have `python` as a `run` requirement, why are you adding it to `ignore_run_exports`?

Reply:
I am not sure what I added this for. I can try to remove it